pkgs/by-name/te/tenv/package.nix

```
@@ -0,0 +1,27 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "tenv";
```

```suggestion
{ buildGoModule, fetchFromGitHub, lib }:

buildGoModule rec {
  pname = "tenv";
```
Add a blank line after parameters.